CHANGELOG.md

```diff
@@ -2,8 +2,20 @@
 
 [Releases](https://github.com/NetApp/harvest/releases)
 
-## 21.08.1 / 2021-??-??
-<!-- git log --no-merges --cherry-pick --right-only --oneline release/21.08.0...release/21.08.1 -->
+## 21.11.0 / 2021-11-08
+<!-- git log --no-decorate --no-merges --cherry-pick --right-only --oneline release/21.08.0...release/21.11.0 -->
```

Comment:
better to use origin before branch name. Else one has to checkout these branches to diff